src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+    result: 0,
+
+    add(value, result) {
+      return result + value;
+    },
+
+    subtract(value, result) {
+      return result - value;
+    },
+
+    multiply(value, result) {
+      return result * value;
+    },
+
+    divide(value, result) {
+      return result / value;
+    },
+
+    reset() {
+      this.result = 0;
```

Comment:
better create constant `RESET_DEFAULT_VALUE` or smth like this instead of hardcoded value(`0`)